modules/core/include/opencv2/core/cv_cpu_dispatch.h

```diff
@@ -168,6 +168,11 @@
 #  include <wasm_simd128.h>
 #endif
 
+#if defined(__riscv)
+#  define CV_RVV 1
+//#  include <riscv_vector.h>
```

Comment:
This header is not needed now for C++ version universal intrinsics and will be needed later for rvv version universal intrinsics. Should I remove it now and add it later when it is needed?

Reply:
Wouldn't it be included in `intrin_riscv.hpp`?

Reply:
I think it might need to be included here. All the other implementation of universal intrinsics included their headers here but not in `intrin_XXX.hpp`. I haven't tested to included it in `intrin_rvv.hpp` but I think it should be consistent with other versions, should it? Then I just remove it at present.